source/backend/tests/unit/api/Services/AcquisitionFileServiceTest.cs

```diff
@@ -1219,8 +1219,126 @@ public void AddCompensationsRequisitions_Success()
             // Assert
             repository.Verify(x => x.Add(It.IsAny<PimsCompensationRequisition>()), Times.Once);
         }
+        #endregion
+
+        #region InterestHolders
+
+        [Fact]
+        public void GetInterestHolders_NoPermissions()
+        {
+            // Arrange
+            var service = CreateAcquisitionServiceWithPermissions();
+
+            // Act
+            Action act = () => service.GetInterestHolders(1);
+
+            // Assert
+            act.Should().Throw<NotAuthorizedException>();
+        }
+
+        [Fact]
+        public void GetInterestHolders_NotAuthorized_Contractor()
+        {
+            // Arrange
+            var service = CreateAcquisitionServiceWithPermissions(Permissions.AcquisitionFileView, Permissions.CompensationRequisitionView);
```

Comment:
nit: don't think this test needs the CompensationRequisitionView permission so would be inclined to remove from test setup